teamcity-rest-client-impl/src/test/kotlin/org/jetbrains/teamcity/rest/BuildTest.kt

```diff
@@ -146,4 +146,24 @@ class BuildTest {
             assertNotNull(iterator.next())
         }
     }
+
+    @Test
+    fun test_parameters() {
+        val build = publicInstance().build(BuildId("699994"))
+        val parameters = build.parameters
+
+        assertTrue(parameters.isNotEmpty())
+        assertEquals(53, parameters.count())
+        assertEquals("1.0.0", parameters.first { it.name == "kotlin.build.number" }.value)
+    }
+
+    @Test
+    fun test_resulting_parameters() {
+        val build = customInstanceByConnectionFile().build(BuildId("699994"))
```

Comment:
To get resulting parameters guest auth isn't enough. I didn't get where connection file is placed, but this test should be working.